src/server/accessibility/index.js

```diff
@@ -0,0 +1,24 @@
+import { Page } from '~/src/server/common/model/page/page-model.js'
+import { PageController } from '~/src/server/common/controller/page-controller/page-controller.js'
+
+export class AccessibilityStatementPage extends Page {
+  sectionKey = 'policies'
+  key = 'accessibility-statement'
+  pageTitle = 'Accessibility statement'
```

Comment:
:D one to extract into a 'const' above - otherwise sonarcloud will let us know all about it.

Reply:
done